pkg/bpf/bpf.go

```diff
@@ -156,14 +156,12 @@ func StopMda() error {
 
 func (l *BpfLoader) Stop() {
 	var err error
-	if restart.GetExitType() == restart.Restart && l.config.DualEngineEnabled() {
-		C.deserial_uninit()
```

Comment:
why remove C.deserial_uninit()

Reply:
The deserial logic is updated, and now the default C.deserial_uninit().